Sources/Noora/Noora.swift

```diff
@@ -57,9 +57,85 @@ public struct ErrorAlert: ExpressibleByStringLiteral {
     }
 }
 
+extension Noorable {
```

Comment:
very minor nit: I would put an extension of the protocol _below_ the definition of that protocol.